dataland-backend/src/main/kotlin/org/dataland/datalandbackend/api/DataApi.kt

```diff
@@ -45,6 +45,7 @@ interface DataApi<T> {
     fun postCompanyAssociatedData(
         @Valid @RequestBody
         companyAssociatedData: CompanyAssociatedData<T>,
+        @RequestParam(defaultValue = "false") bypassQa: Boolean,
```

Comment:
Aus Neugier:
Wenn du keinen default angeben würdest, dann wäre "bypassQa" einfach "null" wenn jmd in seinem request gar keinen request param angibt, oder?
Und wenn du dann "null" genau wie jetzt "false" handlest, hätten wir das gleiche Ergebnis, oder?

Reply:
Da es nicht nullable ist, würde ich eher erwarten, dass eine Antwort mit einen Invalid Input Error zurückgegeben wird.

Reply:
Wir haben beschlossen dass wir es nullable machen, damit die flag nicht in jedem x-beliebigen request zu sehen ist.

Reply:
*fehlendes `bypassQa` wird dann quasi behandelt wie `bypassQa = false`

Reply:
Scheinbar ist eine Änderung gar nicht notwendig. Die Docs für `defaultValue` sagen, dass der default value gesetzt wird, wenn der parameter nicht gesetzt ist.